app/src/org/commcare/network/DataPullResponseFactory.java

```diff
@@ -21,7 +22,7 @@ public RemoteDataPullResponse makeDataPullRequest(DataPullTask task,
                                                       CommcareRequestEndpoints requestor,
                                                       String server,
                                                       boolean includeSyncToken) throws IOException {
-        Response response = requestor.makeCaseFetchRequest(server, includeSyncToken);
+        Response<ResponseBody> response = requestor.makeCaseFetchRequest(server, includeSyncToken);
```

Comment:
is this still relevant ?

Reply:
Yeah, `RemoteDataPullResponse` constructor expects a generic type.